doc/source/examples_source/00-basic/03-ptrace.py

```diff
@@ -0,0 +1,144 @@
+session.ensight.data_partbuild.description("Shuttle")
+session.ensight.data_partbuild.create()
+session.ensight.part.select_byname_begin("(CASE:Case 1)Shuttle")
+session.ensight.case.select("Case 1")
+session.ensight.data_partbuild.end()
+
+
+###############################################################################
+# Setup the View
+# ----------------------------------
+# Set View, Turn on Symmetry, turn off visibility of flow domain.
+
+session.ensight.view_transf.rotate(-109.084335, -1.64276719, 0)
+session.ensight.view_transf.rotate(-2.38553524, 115.462845, 0)
+session.ensight.view_transf.zoom(0.489253074)
+session.ensight.view_transf.look_at(2.4110589, 0.567389309, 0.241451085)
+session.ensight.view_transf.look_from(2.4110589, 0.567389309, 5.69335651)
+
+session.ensight.part.select_all()
+session.ensight.part.modify_begin()
+session.ensight.part.symmetry_type("mirror")
+session.ensight.part.symmetry_mirror_y("ON")
+session.ensight.part.modify_end()
+
+session.ensight.part.select_begin(1)
+session.ensight.part.modify_begin()
+session.ensight.part.visible("OFF")
+session.ensight.part.modify_end()
+
+###############################################################################
+# Create a Particle Trace using the Line Tool Specification
+# ---------------------------------------------------------
+# Using the 3D parts as the parent, with the line tool as the emission type
+# "Momentum" as the vector, and 50 points alone the line as emitter locations.
+
+# .. image:: /_static/03_ptrace_1.png
+
+pt1 = [0.1245, 0.064366, -0.03438]
+pt2 = [1.0018, 0.399756, -0.03258]
+parent_parts = eoutil.parts.select_parts_by_dimension(3)
+npts = 50  # number of emitters
+vector_var = eocore.VARIABLES["Momentum"][0]  # Vector variable to use
+strpart = eoutil.parts.create_particle_trace_from_line(
+    "Streamline",
+    vector_var,
+    point1=pt1,
+    point2=pt2,
+    num_points=npts,
+    source_parts=parent_parts,
+    direction="+/-",
+)
+
+
+###############################################################################
+# Change Visual Attributes
+# ----------------------------------------------------------
+#  Modify the attributes of the Streamline for visual clarity
+
+# .. image:: /_static/03_ptrace_2.png
+
+strpart.REPRESENTATION = eonums.TRACE_TUBE
+strpart.WIDTHSCALEFACTOR = 0.012
+strpart.COLORBYPALETTE = "Momentum"
+
+
+###############################################################################
+# Animate the Streamlines
+# ----------------------------------------------------------
+#  Turn OFF the streamlines (to see the animate under)
+#  Turn ON the animate streamlines.
+#  Change to Sphere representation, size, and adjust speed and length.
+
+# .. image:: /_static/03_ptrace_3.png
+
+strpart.VISIBLE = False
+strpart.ANIMATE = True
+eocore.HEADTYPE = eonums.ATRACE_HEAD_SPHERE
+eocore.HEADSCALE = 0.03
+eocore.PARTICLETIME = 2.0
+eocore.DELTATIME = 0.065
+eocore.MULTIPLEPULSES = True
+
```

Comment:
@kecolburn I am ok with the example, that's good stuff.
I wonder only if we need to add  a line like "session.show("image", width=800, height=600)" at the end of each section where you are describing the output a user should see.
This is because a navigated user that know PyEnSight would probably already know that a renderable needs to be called to actually see the output, but a user X that is drawn to this example because is really interested in just the particle traces would run the code and not see what's happening, having to go to the other examples to understand how to visualize the data.

Reply:
@mariostieriansys I agree. Thanks for catching that. I've made those modifications and will re-submit at new PR.

Reply:
I've made the changes, and attempted to re-commit and re-publish the changes. Not sure I did that right, as I don't see any new PR, or a -re-submit here....